governance.md

```diff
@@ -25,11 +25,12 @@ See [community membership]
 
 # Community groups
 
-The project has 4 main types of groups:
+The project has 5 main types of groups:
 * Special Interest Groups, SIGs
   * Subprojects
```

Comment:
My apologies for using this PR for my question. Regarding`Subprojects`, Is the indent intended? If not, I can raise a PR @cblecker

Reply:
Yes, it is intended. They are subgroups of SIGs. Thanks.